crates/core/component/ibc/src/component/msg_handler/channel_open_ack.rs

```diff
@@ -58,6 +58,7 @@ impl MsgHandler for MsgChannelOpenAck {
             remote: expected_counterparty,
             connection_hops: expected_connection_hops,
             version: self.version_on_b.clone(),
+            ..ChannelEnd::default()
```

Comment:
`ChannelEnd` now has a default impl - so I used it for all new fields. Could set them explicitly?